test/test_from_exception.cpp

```diff
@@ -146,6 +199,8 @@ int main() {
   test_no_exception();
   test_trace_from_exception();
   test_after_other_exception();
+  test_rethrow();
+  test_rethrow_after_other_exception();
   test_nested();
   test_rethrow_nested();
   test_from_other_thread();
```

Comment:
Also change the 

https://github.com/boostorg/stacktrace/blob/351b03d5220510a1b6738cc378659b6d85c8a4da/doc/stacktrace.qbk#L139-L142

to something like

```
 [warning At the moment the functionality is only available for some of the 
          popular C++ runtimes for POSIX systems with *libbacktrace* or for Windows. 
          Make sure that your platform is supported by running some tests. 
 ] 
```